eg/Classes/PluginInstall.py

```diff
@@ -283,3 +431,154 @@ def VisitName(self, node):
 
     def VisitStr(self, node):
         return node.s
+
+
+class HiddenSystemDialog(wx.Dialog):
+
+    def __init__(self, fileFolders):
+
+        self.fileFolders = fileFolders
+        self.root = None
+
+        wx.Dialog.__init__(
+            self,
+            None,
+            title='Plugin Install - Hidden/System Files',
+            size=(450, 525),
+            style=(
+                wx.CAPTION |
+                wx.CLOSE_BOX |
+                wx.SYSTEM_MENU |
+                wx.RESIZE_BORDER |
+                wx.MAXIMIZE_BOX
+            )
+        )
+
+        headerBox = eg.HeaderBox(
+            self,
+            name='Plugin Install',
+            text=(
+                'There are hidden and system files please select\n'
+                'the files you wish to add to the egplugin package.'
```

Comment:
Fantastic

The other PR i did was for adding the more plugin install information.. I will add this to a new class inside of the file and then create a new PR to move all of the other plugin related text out of the Text file that is fi the other PR got merged. i think it did but i honestly do not remember